validate.sh

```diff
@@ -2,9 +2,7 @@
 
 set -eux
 
-docker build . -t parse_a_changelog
-
 docker run \
     --rm \
     -v $PWD/CHANGELOG.md:/CHANGELOG.md \
-    parse_a_changelog
+    parse-a-changelog
```

Comment:
what is the purpose of this script? I can see it is run in the pipeline after build.sh, and it's not documented anywhere - is it supposed to be a sort of local test in the pipeline (in which case it should use the local image), or is it supposed to be validating the changelog of this project (in which case maybe the image should be the dockerhub one)?

Reply:
Added a comment to clarify the purpose, switched to using the dockerhub image and moved the stage to before the build stage as requested [previously](https://github.com/cyberark/parse-a-changelog/pull/17#discussion_r390484316).